install_config/aggregate_logging.adoc

```diff
@@ -953,6 +953,63 @@ Further explanation of how to set up the `fluent-plugin-secure-forward` plug-in
 can be link:https://github.com/tagomoris/fluent-plugin-secure-forward[found
 here].
 
+[[reduce-numer-of-connections-from-fluentd]]
+*Reduce the Number of Connections from Fluentd to the API Server*
+
+With `mux`, you can deploy `N` number of `mux` services, where `N` is fewer than
+the number of nodes. Each Fluentd is configured with `USE_MUX_CLIENT=1`. This
+tells Fluentd to send the raw logs to `mux` with no filtering, and no Kubernetes
+metadata filtering which involves connections to the API server. You can
+perform all of the processing and Kubernetes metadata filtering with `mux`.
+
+[cols="3,7",options="header"]
+|===
+|Parameter
+|Description
+
+| `openshift_logging_use_mux`
+|Default `False`. If set to `True`, a service called `mux` will be deployed. This
+service will act as a Fluentd `secure-forward` tool for the node agent
+Fluentd daemonsets running in the cluster. This can be used to reduce the number
+of connections to the OpenShift API server, by using `mux` and configuring each
+node Fluentd to send raw logs to mux and turn off the Kubernetes metadata plugin.
+
+| `openshift_logging_mux_allow_external`
+|Default `False`. If set to `True`, the `mux` service will be deployed, and it will be
+configured to allow Fluentd clients running outside of the cluster to send logs
+using `secure-forward`. This allows OpenShift logging to be used as a central
+logging service for clients other than OpenShift, or other OpenShift clusters.
+
+| `openshift_logging_use_mux_client`
+|Default `False`. If this is `True`, the node agent Fluentd services will be
+configured to send logs to the `mux` service rather than directly to
+Elasticsearch.
```

Comment:
`, and their Kubernetes metadata filters will be disabled, reducing the number of connections to the API server.`